WalletWasabi.Gui/Assets/LegalDocuments.txt

```diff
@@ -169,8 +169,8 @@ Personally Identifiable Information
 Handling information
 ====================
 
-Since we are working on privacy enhancement projects and our mission is to regain personal privacy, our services are designed to be used without indication of any personal data. For this reason we do not have Google Analytics or any other kind of data collecting solutions built into our website. This means that we do not even have any knowledge of the website’s traffic. Visitors may, however, indicate their email addresses voluntarily to get notifications in case of any potential technical problems or other enquiries. These e-mail addresses are solely used to answer users’ questions and are erased after 100 days.
-As the main platform for users’ technical questions and issues we use Reddit to answer all questions.
+Since we are working on privacy enhancement projects and our mission is to regain personal privacy, our services are designed to be used without indication of any personal data. For this reason we do not have Google Analytics or any other kind of data collecting solutions built into our website. This means that we do not even have any knowledge of the website’s traffic. Visitors may, however, indicate their email addresses voluntarily to get notifications in case of any potential technical problems or other inquiries. These e-mail addresses are solely used to answer users’ questions and are erased after 100 days.
+We use Reddit as the main platform for users’ technical questions and issues, and we do not retain any data that can be subsequently identified / associated with the user.
```

Comment:
This is a new addition, not just a typo / grammar fix.
I think it is good to add, but please review.

Reply:
Yeah, I didn't consider it an addition as much as a clarification of the privacy policy that more clearly addressed people's use of Reddit to communicate with support. If anything, it's redundant, but generally in legal documents, redundancy can be a good thing.
If anyone on the Wasabi team has a problem with it, I'm happy to remove it.